PYIC-4572: Update RevokeVcsHandler and RestoreVcsHandler READMEs (#1603)
alphagov/di-ipv-core-back: lambdas/revoke-vcs/README.md

- In order to safely revoke these VCs we perform the following operation on each VC:
  1. Read the VC from the main table, fails if not exists.
     - The `userId` and `criId` keys provided in each entry of the payload list uniquely identify a VC.
  2. Enter VC into the archive table.
  3. Submit audit event `IPV_VC_REVOKED` for the revocation.
  4. Delete the VC from the main table.
  5. Log the progress through the list.

- This lambda works on user credentials in the `user-issued-credentials-v2-ENVIRONMENT` table ([e.g.](https://eu-west-2.console.aws.amazon.com/dynamodbv2/home?region=eu-west-2#item-explorer?table=user-issued-credentials-v2-build)).
- The lambda archives during revocation in the `revoked-user-credentials-ENVIRONMENT` table ([e.g.](https://eu-west-2.console.aws.amazon.com/dynamodbv2/home?region=eu-west-2#item-explorer?table=revoked-user-credentials-build)).
- In the case of failures which can have audit events, we send the audit event `IPV_VC_REVOKED_FAILURE`.
- In the case where we cannot send audit events, we break the loop through the list.

---

## Running the lambda

> > This lambda should not be run by anyone without the explicit sign-off of stakeholders.

Reviewer: ```suggestion
> This lambda should not be invoked in production without explicit sign-off from stakeholders
```
